casper/src/test/scala/coop/rchain/casper/genesis/contracts/TestSetUtil.scala

```diff
@@ -34,13 +38,18 @@ object TestSetUtil {
   def runTests(tests: Par, otherLibs: Seq[Par], runtime: Runtime)(
       implicit scheduler: Scheduler): Unit = {
     //load "libraries" required for all tests
-    eval_term(LinkedList.term, runtime)
-    eval_term(TestSet.term, runtime)
+    val bytes = new Array[Byte](128)
+    new SecureRandom().nextBytes(bytes)
+    val rand = Blake2b512Random(bytes)
```

Comment:
If you add the smart constructor to the `Blake2b512Random` refactor this part as well.

Reply:
Done.